src/dynatrace/opentelemetry/metrics/export/_exporter.py

```diff
@@ -30,8 +31,12 @@
 )
```

Comment:
it seems like PyCharm consistently picks `_internal.aggregation` over `export`. :slightly_frowning_face: